content/home.md

```diff
@@ -194,22 +191,22 @@ Try Quire for yourself
 
 ![map icon](/img/icons/map.png)
 
-### Product Roadmap
+### Community Forum
 
-Find out what’s coming coming next
+Ask and answer questions
 
-[See Quire roadmap](/about/roadmap/)
+[Join the conversation](/community/forum/)
 
 </div>
 <div class="footer-item">
 
 ![document icon](/img/icons/document.png)
 
-### Documentation
+### Contact Us
```

Comment:
I think Documentation is better here because it potentially gives somewhere for Developers to go, and I think "Contact us" and "Stay in Touch" are too similar.